### ISSUE TITLE
How to Load SNPs information into higlass?

### ISSUE THREAD

**Issue by lhqing** [higlass/higlass#717]
Hi, thank you for building this wonderful project!

I have a question about load SNP information into higlass 1D tracks, and I hope those SNP can have labels shown close to it when zooming in. I did some search and found that the plugin [higlass-labelled-points-track](https://github.com/higlass/higlass-labelled-points-track) may be suitable for this. However, this seems that we need to write our own JS code to load that plugin.

I wonder if there is a way to load SNP information into higlass using higlass docker & higlass-manage command without writing JS? and what would be the input file format?

Thank you very much!

**Comment by pkerpedjiev** [comment]
Hey, sorry for the delayed response! What format is your SNP data in?

If it's a bed file, you should be able to load it using the following commands:

```
clodius aggregate your_bed_file.bed --chromsizes-filename your_assembly.chrom.sizes
higlass-manage view your_bed_file.bed.beddb
```

This won't show you labels above the SNP's but we have code in a branch that will let you do that. It should make it's way into a release within the next month.

**Comment by pkerpedjiev** [comment]
To make sure this doesn't get lost, here's the PR: https://github.com/higlass/higlass/pull/731

**Comment by lhqing** [comment]
Thank you @pkerpedjiev for your reply! 
The typical SNP table is in VCF format, but I can change it to bed format by myself.

**Comment by pkerpedjiev** [comment]
Yeah, sorry we don't support VCF just yet. Hopefully we'll get to it in the not too distant future :-) Please let me know if you have any issues with bed files.
